cmd/podman/history.go

```diff
@@ -158,18 +159,23 @@ func (h *historyTemplateParams) headerMap() map[string]string {
 }
 
 // getHistorytemplateOutput gets the modified history information to be printed in human readable format
-func getHistoryTemplateOutput(history []v1.History, layers []types.BlobInfo, imageID string, opts historyOptions) (historyOutput []historyTemplateParams) {
+func getHistoryTemplateOutput(history []v1.History, layers []types.BlobInfo, imageIDs []string, opts historyOptions) (historyOutput []historyTemplateParams) {
 	var (
 		outputSize  string
 		createdTime string
 		createdBy   string
+		imageID     string
 		count       = 1
+		imgIDCount  = 0
 	)
 	for i := len(history) - 1; i >= 0; i-- {
-		if i != len(history)-1 {
-			imageID = "<missing>"
+		if imgIDCount < len(imageIDs) {
+			imageID = imageIDs[imgIDCount]
+			imgIDCount++
+		} else {
+			imageID = "<none>"
```

Comment:
For the sake of docker consistency, should this (and related changes) remain as `<missing>`?

Reply:
Yup, fixed. Not sure why I thought it was none...